openedx/core/djangoapps/edx_global_analytics/tests/test_acceptor_api_usage.py

```diff
@@ -0,0 +1,173 @@
+"""
+Tests acceptor API usage by edx global analytics application functions aka utils.
+"""
+
+import uuid
+import unittest
+
+from mock import patch
+
+from openedx.core.djangoapps.edx_global_analytics.utils import send_instance_statistics_to_acceptor
+from openedx.core.djangoapps.edx_global_analytics.token_utils import (
+    access_token_authorization,
+    access_token_registration,
+    get_acceptor_api_access_token,
+)
+
+
+@patch('openedx.core.djangoapps.edx_global_analytics.utils.requests.post')
+class TestAcceptorApiUsage(unittest.TestCase):
+    """
+    Test acceptor API usage by edx global analytics application functions aka utils.
+    """
+
+    def test_token_registration(self, mock_request):
+        """
+        Verify that access_token_registration sends request to acceptor API for token registration.
+        """
+        access_token_registration('https://mock-url.com')
+
+        mock_request.assert_called_once_with('https://mock-url.com' + '/api/token/registration/')
```

Comment:
Are you testing here some url again? (like that url is /api/token/registration/)? If so , I this it is not very useful.